test/3x/common/test_logger.py

```diff
@@ -0,0 +1,68 @@
+"""Tests for common logger.
+
+!!! Please do not import any framework-specific modules in this file. !!!
+* Note, we may need to add some auto check mechanisms to ensure this.
+
+These tests aim to assess the fundamental functionalities of common components and enhance code coverage.
+All tests will be included for each framework CI.
+"""
+
+import unittest
+
+from neural_compressor.common.utils import Logger
+
+log_msg_lst = [
+    "call logger log function.",
+    {"msg": "call logger log function."},
+    ["call logger warning function", "done"],
+    ("call logger warning function", "done"),
+    # the following log will be prettified
+    {"msg": "call logger warning function"},
+    {"msg": {("bert", "embedding"): {"weight": {"dtype": ["unint8", "int8"]}}}},
+    {"msg": {("bert", "embedding"): {"op": ("a", "b")}}},
+    # the following log will not be prettified
+    [{"msg": "call logger warning function"}, {"msg2": "done"}],
+    ({"msg": "call logger warning function"}, {"msg2": "done"}),
+    ({"msg": [{"sub_msg": "call logger"}, {"sub_msg2": "call warning function"}]}, {"msg2": "done"}),
+]
+
+
+class TestLogger(unittest.TestCase):
```

Comment:
Improvements can be made to ensure that the exception handling works, such as passing empty messages and exceptional messages at different levels.

Reply:
The Logger invokes the built-in `logging` package directly, which  should have a comprehensive test.